src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/graphs/KBestHaplotypeFinderUnitTest.java

```diff
@@ -265,7 +266,7 @@ public void testBasicBubbleData(final int refBubbleLength, final int altBubbleLe
         expectedCigar.add(new CigarElement(postRef.length(), CigarOperator.M));
 
         final String ref = preRef + v2Ref.getSequenceString() + postRef;
-        Assert.assertEquals(path.calculateCigar(ref.getBytes()).toString(), AlignmentUtils.consolidateCigar(expectedCigar).toString(), "Cigar string mismatch");
+        Assert.assertEquals(path.calculateCigar(ref.getBytes(), SmithWatermanJavaAligner.getInstance()).toString(), AlignmentUtils.consolidateCigar(expectedCigar).toString(), "Cigar string mismatch");
```

Comment:
Do we need to refactor these tests so they run with both aligners (when we have them). Maybe that should be a separate ticket/PR ?

Reply:
I wasn't sure.  I felt like they're really designed to test something else and hopefully the unit test for the aligners is good, but it wouldn't be bad to run them with all aligners just be sure.